macos/uvkc-cpu.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: uvkc-cpu
+  namespace: default
+spec:
+  containers:
+  - name: vkmark
```

Comment:
Need to fix the name, leftover from the vkmark benchmark that cannot run with our virtio-gpu.